src/uniprotkb/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,286 @@
+    title: 'How to explore a UniProt entry',
+    date: '17 Feb 2022',
+  },
+  {
+    id: 'p4_gGkM-Rfs',
+    title: 'How to download embeddings in UniProt',
+    date: '25 Jul 2023',
+  },
+];
+
+const LandingPage = () => {
+  const release = useUniProtDataVersion();
+
+  const { data } = useDataApi<SearchResults<never>>(
+    getAPIQueryUrl({
+      namespace: Namespace.uniprotkb,
+      query: '*',
+      size: 0,
+      facets: ['reviewed'],
+    })
+  );
+
+  let numberReviewed: undefined | number;
+  let numberUnreviewed: undefined | number;
+
+  if (data?.facets?.[0].values?.length) {
+    numberReviewed = data?.facets?.[0].values.find(
+      (value) => value.value === 'true'
+    )?.count;
+    numberUnreviewed = data?.facets?.[0].values.find(
+      (value) => value.value === 'false'
+    )?.count;
+  }
+
+  return (
+    <div className={styles['landing-page']}>
+      <h2 className={styles['landing-page__title']}>UniProtKB</h2>
+      <div className={cn('uniprot-grid', styles['landing-page__content'])}>
+        <div
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            'uniprot-grid-cell--medium-span-4'
+          )}
+        >
+          <YouTubeEmbed id="OwOJmKmc7VM" title="Welcome to UniProt" />
+        </div>
+        <p
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            'uniprot-grid-cell--medium-span-8'
+          )}
+        >
+          The UniProt Knowledgebase (UniProtKB) is the central hub for the
+          collection of functional information on proteins, with accurate,
+          consistent and rich annotation. In addition to capturing the core data
+          mandatory for each UniProtKB entry (mainly, the amino acid sequence,
+          protein name or description, taxonomic data and citation information),
+          as much annotation information as possible is added. <br />
+          <br />
+          The UniProt Knowledgebase consists of two sections: a section
+          containing manually-annotated records with information extracted from
+          literature and curator-evaluated computational analysis
+          (UniProtKB/Swiss-Prot), and a section with computationally analyzed
+          records that await full manual annotation (UniProtKB/TrEMBL).
+          <Link
+            to={{
+              pathname: LocationToPath[Location.UniProtKBResults],
+              search: `query=*`,
+            }}
+            className={styles['search-link']}
+          >
+            Start searching in UniProtKB <big>»</big>
+          </Link>
+        </p>
+
+        {/* Statistics */}
+        <div
+          className={cn(
+            'uniprot-grid-cell--small-span-12',
+            'uniprot-grid-cell--medium-span-9',
+            styles.statistics
+          )}
+        >
+          <h4>Statistics</h4>
+          <div className={styles.statistics__content}>
+            <div className={styles.chart}>
+              <Link
```

Comment:
This link is inline, so if the page is too wide it will go on the left of the graph. This should either be placed within a div (that will be block) or be added a class to assign `display: block` to the link itself

Reply:
Making it a heading could also work, and would be OK for both this and the "Number of entries" text in the next grid cell